gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagManager.java

```diff
@@ -411,6 +416,11 @@ public synchronized void setActive(boolean active) {
     }
   }
 
+  @VisibleForTesting
+  protected static MetricNameRegexFilter getMetricsFilterForDagManager() {
```

Comment:
minor: since part of the `DagManager` class seems redundant to add suffix, `ForDagManager`.

(e.g. look how it appears when called in the test)